web/impact/impact/tests/test_graphql.py

```diff
@@ -510,35 +510,40 @@ def test_query_program_roles_for_expert_with_same_program(self):
         self._assert_response_equals_json(query, expected_json)
 
     def test_query_prg_roles_for_selected_roles(self):
```

Comment:
This test is very obscure - the name does not tell me anything about what it's meant to be testing, which _seems_ to be that a staff user gets a certain bundle of data back from this function `get_user_program_and_startup_roles`, and also gets the same bundle of data back from graphql, which by a coincidence, happens to call that function. 
I'm not sure that's a very interesting test, tbh, but clearly we need a test for the functionality change: non-staff users should not get back the entrant labels. 
We should also at least improve the naming of the test. 

If we can do this without adding (even more) duplication in this set of tests, that would be lovely.